src/Nethermind/Nethermind.Consensus.Clique/CliquePlugin.cs

```diff
@@ -192,5 +192,21 @@ public Task InitRpcModules()
         private ICliqueConfig? _cliqueConfig;
 
         private IBlocksConfig? _blocksConfig;
+        public IModule? GetModule(string engineType, IConfigProvider configProvider)
+        {
+            if (engineType != SealEngineType) return null;
```

Comment:
Would be good to have this Guard somehow integrated into the design and not manual

Reply:
Ok